base/src/main/java/com/austindroids/austinfeedsme/eventsfilter/EventFilterActivity.kt

```diff
@@ -0,0 +1,41 @@
+package com.austindroids.austinfeedsme.eventsfilter
+
+import android.os.Bundle
+import com.austindroids.austinfeedsme.R
+import com.austindroids.austinfeedsme.common.base.BaseActivity
+import com.austindroids.austinfeedsme.data.Event
+import com.austindroids.austinfeedsme.data.FilterableEventsRepository
+import kotlinx.android.synthetic.main.activity_event_filter.*
+import javax.inject.Inject
+
+/**
+ * Created by darrankelinske on 8/4/16.
+ */
+class EventFilterActivity : BaseActivity(), EventFilterContract.View {
+
+    @Inject
+    internal lateinit var eventFilterPresenter: EventFilterPresenter
+    @Inject
+    internal lateinit var eventsRepository: FilterableEventsRepository
+
+    private var eventFilterAdapter: EventFilterAdapter? = null
```

Comment:
You create this adapter right in `onCreate` so its probably safe to make it non-nullable and `lateinit`, then you don't have to add null checks everywhere